src/test/scala/com/campudus/tableaux/api/structure/SettingsTableTest.scala

```diff
@@ -147,4 +147,104 @@ class SettingsTableTest extends TableauxTestBase {
         test <- sendRequest("DELETE", s"/tables/$tableId/rows/1")
       } yield assertEquals(expectedOkJson, test)
     }
+
+  @Test
+  def insertKeyIntoSettingsTable(implicit c: TestContext): Unit =
+    okTest {
+      def settingsRow = Json.obj(
+        "columns" -> Json.arr(Json.obj("id" -> 1)),
+        "rows" -> Json.arr(Json.obj("values" -> Json.arr("already_existing_key")))
```

Comment:
Ich meinte das anders. Hier in dem Test macht `already_existing_key` keinen Sinn, oder?